src/main/java/uk/gov/hmcts/reform/idam/web/config/IdamWebMvcConfiguration.java

```diff
@@ -75,6 +85,16 @@ public RequestMethodInterceptor requestMethodInterceptor() {
         return new RequestMethodInterceptor();
     }
 
+    @Bean
+    public RequestRejectedHandler requestRejectedHandler() {
+        return new HttpStatusRequestRejectedHandler(HttpStatus.BAD_REQUEST.value()) {
+            @Override
+            public void handle(HttpServletRequest request, HttpServletResponse response, RequestRejectedException requestRejectedException) throws IOException {
+                log.error("Request rejected due to Spring Security - ", requestRejectedException);
```

Comment:
the dash at the end won't make any sense, there'll be nothing after it in the "message" attribute.

Reply:
Ooops, fixed!